hawkbit-rest/hawkbit-mgmt-resource/src/test/java/org/eclipse/hawkbit/mgmt/rest/resource/MgmtRolloutResourceTest.java

```diff
@@ -1043,4 +1045,57 @@ amountGroups, new RolloutGroupConditionBuilder().withDefaults()
 
         return rolloutManagement.get(rollout.getId()).orElseThrow(NoSuchElementException::new);
     }
+
+    @Test
+    @Description("Testing that pausing the rollout switches the state to paused")
+    void triggeringNextGroupRollout() throws Exception {
+        // setup
+        final int amountTargets = 20;
+        testdataFactory.createTargets(amountTargets, "rollout", "rollout");
+        final DistributionSet dsA = testdataFactory.createDistributionSet("");
+
+        // create rollout including the created targets with prefix 'rollout'
+        final Rollout rollout = createRollout("rollout1", 4, dsA.getId(), "controllerId==rollout*");
+
+        // starting rollout
+        rolloutManagement.start(rollout.getId());
+
+        // Run here, because scheduler is disabled during tests
+        rolloutManagement.handleRollouts();
+
+        // trigger next group on the rollout
+        mvc.perform(post("/rest/v1/rollouts/{rolloutId}/triggerNextGroup", rollout.getId())).andDo(MockMvcResultPrinter.print())
+                .andExpect(status().isOk());
+
+        // check rollout is in running state
+        mvc.perform(get("/rest/v1/rollouts/{rolloutId}", rollout.getId()).accept(MediaType.APPLICATION_JSON))
+                .andDo(MockMvcResultPrinter.print()).andExpect(status().isOk())
+                .andExpect(content().contentType(MediaType.APPLICATION_JSON_VALUE))
+                .andExpect(jsonPath("id", equalTo(rollout.getId().intValue())))
+                .andExpect(jsonPath("status", equalTo("running")));
+
+        final RolloutGroup firstGroup = rolloutGroupManagement
+                .findByRollout(PageRequest.of(0, 1, Direction.ASC, "id"), rollout.getId()).getContent().get(0);
+        final RolloutGroup secondGroup = rolloutGroupManagement
+                .findByRollout(PageRequest.of(1, 1, Direction.ASC, "id"), rollout.getId()).getContent().get(0);
+
+
+        retrieveAndVerifyRolloutGroupInRunning(rollout, firstGroup);
```

Comment:
There is no need to make rest calls for that. Just check if firstGroup.getStatus() and secondGroup .getStatus() has the expected value